src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './App/App.js';
```

Comment:
Perfect. Having each component in its own folder will make this so much easier to read and work with.